File: javascript-closure-angular-client/API/Client/PaymentTransactionOut.js
goog.provide('API.Client.PaymentTransactionOut');

/**
 * @record
 */
API.Client.PaymentTransactionOut = function() {}

/**
 * @type {!API.Client.PayRecHeader}
 * @export
 */
API.Client.PaymentTransactionOut.prototype.header;

/**
 * @type {!Array<!API.Client.PayRecLinesOut>}
 * @export
 */
API.Client.PaymentTransactionOut.prototype.lines;

/**
 * @type {!API.Client.PayRecCalculatedTaxSummaryForService}
 * @export
 */
API.Client.PaymentTransactionOut.prototype.calculatedTaxSummary;

/**
 * @type {!API.Client.ProcessingInfo}
 * @export
 */
API.Client.PaymentTransactionOut.prototype.processingInfo;

File: async-scala-client/src/main/scala/io/swagger/client/model/PaymentTransactionOut.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PaymentTransactionOut (
  header: Option[PayRecHeader],
lines: Option[List[PayRecLinesOut]],
calculatedTaxSummary: Option[PayRecCalculatedTaxSummaryForService],
processingInfo: Option[ProcessingInfo]
)
